main.nf

```diff
@@ -30,6 +30,7 @@ def helpMessage() {
      --genome [str]                     Name of iGenomes reference
      --single_end [bool]                Specifies that the input is single end reads
      --comprehensive [bool]             Output information for all cytosine contexts
+     --cytosine_report [bool]           Output stranded cytosine report during Bismark's bismark_methylation_extractor step.
```

Comment:
We should also add an entry for this in [parameters.settings.json](https://github.com/nf-core/methylseq/blob/master/parameters.settings.json) and in [CHANGELOG.md](https://github.com/nf-core/methylseq/blob/dev/CHANGELOG.md)

Reply:
Done in https://github.com/nf-core/methylseq/pull/137